core/lib/brightcontent/view_lookup/base.rb

```diff
@@ -7,7 +7,7 @@ class Base
 
       def initialize(view_context, options)
         @view_context = view_context
-        @options = options
+        @options = options.symbolize_keys
```

Comment:
Well, it does work without this too, yes. It's a change I found necessary to reliably fetch options using symbols in FilterField. I would not want to deliver code that makes an assumption while that can easily be avoided.